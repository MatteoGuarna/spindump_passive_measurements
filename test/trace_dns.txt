A number of DNS queries
