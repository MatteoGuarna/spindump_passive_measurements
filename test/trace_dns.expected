DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 54423:53 at 1553422217239860 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 61426:53 at 1553422217239918 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 54423:53 at 1553422217281911 measurement starting right 42051 packets 1 0 bytes 121 0 bandwidth 121 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 61426:53 at 1553422217281913 measurement starting right 41995 packets 1 0 bytes 121 0 bandwidth 121 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 51287:53 at 1553422217284471 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 63443:53 at 1553422217284579 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 58598:53 at 1553422217505499 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 50480:53 at 1553422217506105 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 51287:53 at 1553422217519037 measurement starting right 234566 packets 1 0 bytes 97 0 bandwidth 97 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 63443:53 at 1553422217519040 measurement starting right 234461 packets 1 0 bytes 97 0 bandwidth 97 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 49875:53 at 1553422217520323 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 49875:53 at 1553422217557933 measurement starting right 37610 packets 1 0 bytes 91 0 bandwidth 91 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 50480:53 at 1553422217660062 measurement starting right 153957 packets 1 0 bytes 76 0 bandwidth 76 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 58598:53 at 1553422217660499 measurement starting right 155000 packets 1 0 bytes 76 0 bandwidth 76 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 61906:53 at 1553422221634920 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 61906:53 at 1553422222213709 measurement starting right 578789 packets 1 0 bytes 74 0 bandwidth 74 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 49638:53 at 1553422228700147 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 49638:53 at 1553422228784732 measurement starting right 84585 packets 1 0 bytes 85 0 bandwidth 85 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 55623:53 at 1553422235977925 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 55623:53 at 1553422236002004 measurement starting right 24079 packets 1 0 bytes 76 0 bandwidth 76 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 63318:53 at 1553422236539115 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 63318:53 at 1553422236541953 measurement starting right 2838 packets 1 0 bytes 88 0 bandwidth 88 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 52131:53 at 1553422242569000 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 52131:53 at 1553422242575988 measurement starting right 6988 packets 1 0 bytes 73 0 bandwidth 73 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 60967:53 at 1553422249154129 new starting packets 0 0 bytes 0 0
DNS 2001:67c:1232:144:9498:6df6:f450:110b <-> 2001:67c:370:229::7 60967:53 at 1553422249605960 measurement starting right 451831 packets 1 0 bytes 75 0 bandwidth 75 0
